src/index.ts

```diff
@@ -35,25 +35,36 @@ export async function load(base = BASE_PATH, options = { size: IMAGE_SIZE }) {
   return nsfwnet
 }
 
+interface IOHandler {
+  load: () => any
+}
+
 export class NSFWJS {
   public endpoints: string[]
 
   private options: nsfwjsOptions
-  private path: string
+  private pathOrIOHandler: string
   private model: tf.LayersModel
   private intermediateModels: { [layerName: string]: tf.LayersModel } = {}
 
   private normalizationOffset: tf.Scalar
 
-  constructor(base: string, options: nsfwjsOptions) {
+  constructor(
+    modelPathBaseOrIOHandler: string | IOHandler,
+    options: nsfwjsOptions
+  ) {
     this.options = options
-    this.path = `${base}model.json`
     this.normalizationOffset = tf.scalar(255)
+
+    if (typeof modelPathBaseOrIOHandler === 'string') {
+      this.pathOrIOHandler = `${modelPathBaseOrIOHandler}model.json`
+    } else {
```

Comment:
this line is a bit confusing.  Can you delete it?

Reply:
Nope, this was supposed to set `pathOrIOHandler` in the case of `IOHandler`

Reply:
lol, I was tired too, bc I didn't notice it wasn't set, hehehe.